containerized-tests.groovy

```diff
@@ -30,7 +30,7 @@ node('cico-workspace') {
 
 	try {
 		stage('prepare bare-metal machine') {
-			if ("${ghprbPullId}".length() != 0) {
+			if (params.ghprbPullId != null) {
```

Comment:
Will it work even if ghprbPullId is not set?

Reply:
I do not think it will be possible to be empty anymore, once remove from the parameters in the `.yaml` file. The github-pull-request builder plugin will set it to a number of the PR, or it will not be set in case the job is started an other way.

Reply:
Great!

Reply:
yes, it should, that is the whole idea :)